crates/hotshot/src/traits/networking/memory_network.rs

```diff
@@ -187,12 +180,7 @@ impl<TYPES: NodeType> TestableNetworkingImplementation<TYPES>
         Box::pin(move |node_id| {
             let privkey = TYPES::SignatureKey::generated_from_seed_indexed([0u8; 32], node_id).1;
             let pubkey = TYPES::SignatureKey::from_private(&privkey);
-            let net = MemoryNetwork::new(
-                pubkey,
-                NetworkingMetricsValue::default(),
-                Arc::clone(&master),
-                reliability_config.clone(),
-            );
+            let net = MemoryNetwork::new(pubkey, &Arc::clone(&master), reliability_config.clone());
```

Comment:
I only changed this because Rust had updated and thrown me a compile error for the `Arc` not getting consumed in the function body. Let me see if I can improve it quickly